spec/lib/facilities/ppms/v1/response_spec.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+require 'facilities/ppms/v1/response'
+
+describe Facilities::PPMS::V1::Response, team: :facilities do
+  describe '.from_provider_locator' do
+    it 'creates Providers from a ppms response' do
```

Comment:
Are you planning on adding examples here?

Reply:
Maybe?